index.html

```diff
@@ -175,8 +175,8 @@ <h1>Visit Shop Today</h1>
     <div class="footer-row">
         <div class="footer-left">
             <h1>Opening Hours</h1>
-            <p>Monday To Friday-  9am to 9pm</p>
-            <p>Saturday and Sunday - 8am to 11am</p>
+            <p>Monday To Friday - 7am to 9pm</p>
+            <p>Saturday and Sunday - 9am to 12pm</p>
```

Comment:
why the change?

Reply:
making sure the time is at clients benefits

Reply:
got it